superset/config.py

```diff
@@ -163,6 +163,7 @@
     'zh': {'flag': 'cn', 'name': 'Chinese'},
     'ja': {'flag': 'jp', 'name': 'Japanese'},
     'de': {'flag': 'de', 'name': 'German'},
+    'pt-BR': {'flag': 'pt-BR', 'name': 'Portuguese/Brazil'},
```

Comment:
Looking at the other language, shouldn't this be `'pt': {flag: 'br', 'name': 'Brazilian portugese'}`?